activerecord/test/cases/adapters/postgresql/uuid_test.rb

```diff
@@ -125,6 +135,8 @@ def test_acceptable_uuid_regex
      # so we shouldn't block it either. (Pay attention to "fb6d" – the "f" here
      # is invalid – it must be one of 8, 9, A, B, a, b according to the spec.)
      "{a0eebc99-9c0b-4ef8-fb6d-6bb9bd380a11}",
+     # Support Object-Oriented UUIDs which respond to #to_s
+     DuckUUID.new("A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A11"),
     ].each do |valid_uuid|
       uuid = UUIDType.new guid: valid_uuid
       assert_not_nil uuid.guid
```

Comment:
Can you change the assertion to ensure returning string as before?
```suggestion
      assert_instance_of String, uuid.guid
```

Reply:
This is an improvement I can get behind.

Reply:
Ah! It caused another bug because I also implemented the DuckUUID compatability...